cmd/frcli/utils.go

```diff
@@ -274,3 +275,18 @@ func parseChannelPoint(ctx *cli.Context) (*wire.OutPoint, error) {
 
 	return channelPoint, nil
 }
+
+// parseFiatBackend parses the user chosen fiat backend into a FiatBackend type.
+func parseFiatBackend(fiatBackend string) (frdrpc.FiatBackend, error) {
```

Comment:
Ah I see why you went with strings now! Still got some pref for an enum with a `String` method, then this could be 

```
switch fiatBackend{
   case fiat.CoinCapPriceBackend.String(), "":
        return frdrpc.FiatBackend_COINCAP, nil

   case CoinDeskPriceBackend.String():
         return frdrpc.FiatBackend_COINDESK, nil
  
   default: 
       err unknown backend
}
```